### PR TITLE
Remove gesture event mixin

### PR DESCRIPTION
**Pull request by katejeffreys**

### CODE REVIEW

**Comment by katejeffreys** [comment]
@noahlaux this replaces your PR, thanks again

**Comment by serixscorpio** [comment]
@katejeffreys Thanks for this update, I found it helpful! I was using `Polymer.GestureEventListeners` mixin before

**Comment by katejeffreys** [comment]
@arthurevans

File: app/2.0/start/first-element/step-4.md [hunk 1]
```diff
@@ -5,72 +5,38 @@ subtitle: "Build your first Polymer element"
...
-When working with gesture events, you use the `Polymer.Gestures.addListener` method in place of the
-standard `addEventListener` method.
-
-Add a method to toggle the `pressed` property when the button is pressed. Place it inside the class
-definition for `IconToggle`, after the constructor.
+Now, create the `toggle` method to toggle the `pressed` property when the button is pressed. Place the `toggle` method inside the class definition for `IconToggle`, after the constructor.
 
 icon-toggle.html { .caption }
 
 ```html
```
Comment (arthurevans) [inline app/2.0/start/first-element/step-4.md]: this should be highlighted as js, not html

**Pull request merged**